File: Competitions/UC/Berkeley-2024/contest/contest/literally1984/templates/literally1984_template.cpp
#include <iostream>
#include <vector>

using namespace std;

/**
 * Return a length 2 vector containing the coordinates x and y for the given address N.
 * 
 * N: a positive integer, the address of your house
 */
vector<int> solve(int N) {
    // YOUR CODE HERE
    return vector<int>({-1, -1});
}

int main() {
    int T;
    cin >> T;
    for (int i = 0; i < T; i++) {
        int N;
        cin >> N;
        vector<int> ans = solve(N);
        cout << ans[0] << ' ' << ans[1] << '\n';
    }
}

File: Competitions/UC/Berkeley-2024/contest/contest/literally1984/templates/literally1984_template.java
import java.io.*;
import java.util.*;

class Solution {
    /**
     * Return a length 2 List containing the coordinates X and Y.
     * 
     * N: a positive integer, the address of your house
     */
    static List<Integer> solve(int N) {
        // YOUR CODE HERE
        
        return Arrays.asList(-1, -1);
    }
    
    static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
    static PrintWriter out = new PrintWriter(System.out);

    public static void main(String[] args) throws IOException {
        int T = Integer.parseInt(in.readLine());
        for (int i = 0; i < T; i++) {
            String[] temp = in.readLine().split(" ");
            int N = Integer.parseInt(temp[0]);
            List<Integer> ans = solve(N);
            out.println(ans.get(0) + " " + ans.get(1));
        }
        out.flush();
    }
}
